tasks/orca_catalog_reporting_dummy/schemas/output.json

```diff
@@ -1,111 +1,122 @@
-												"fileSizeBytes": {
-													"description": "Size in bytes of the file. From Cumulus ingest.",
-													"type": "integer"
-												},
-												"fileChecksum": {
-													"description": "Checksum hash of the file provided by Cumulus.",
-													"type": "string"
-												},
-												"fileChecksumType": {
-													"description": "Hash type used to calculate the hash value of the file.",
-													"type": "string"
-												},
-												"fileVersion": {
-													"description": "AWS provided version of the file.",
-													"type": "string"
-												},
-												"orcaEtag": {
-													"description": "AWS provided etag of the versioned file. Semi unique.",
-													"type": "string"
-												}
-											},
-											"required": [
-												"fileName",
-												"orcaLocation",
-												"fileSizeBytes",
-												"fileChecksum",
-												"fileChecksumType",
-												"fileVersion",
-												"orcaEtag"
-											]
-										}
-									}
-								},
-								"required": [
-									"granuleId",
-									"date",
-									"createDate",
-									"lastUpdate",
-									"files"
-								]
-							}
-						}
-					},
-					"required": ["collectionId", "granules"]
-				}
-			}
-		},
-		"required": [
-			"providerId",
-			"collections"
-		]
-	}
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/catalog_reporting/schemas/output.json",
+  "title": "catalog_reporting Lambda Output",
+  "description": "The output for the catalog_reporting Lambda.",
+  "type": "object",
+  "required": [
+    "anotherPage",
+    "granules"
+  ],
+  "properties": {
+    "anotherPage": {
+      "description": "Indicates if more results can be retrieved on another page.",
+      "type": "boolean"
+    },
+    "granules": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "required": [
+          "providerId",
+          "collectionId",
+          "id",
+          "createdAt",
+          "executionId",
+          "ingestDate",
+          "lastUpdate",
+          "files"
+        ],
+        "properties": {
+          "providerId": {
+            "description": "The unique ID of the provider. Provided by Cumulus.",
+            "type": "string"
+          },
+          "collectionId": {
+            "description": "The unique ID of the collection. Provided by Cumulus.",
+            "type": "string"
+          },
+          "id": {
+            "description": "The unique ID of the granule. Provided by Cumulus.",
+            "type": "string"
+          },
+          "createdAt": {
+            "description": "Date and time data was originally ingested into cumulus.",
+            "type": "string",
+            "pattern": "^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]*)?Z"

```

Comment:
I think we decided to use this pattern last time
`^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1-9]|[12][0-9])T(2[0-3]|[01][0-9]):([0-5][0-9]):([0-5][0-9])(.[0-9]+)?(Z)?\\+00:00$`

Reply:
This one might be a bit trickier, but I still think we'll need to find a way to match Cumulus' format.